bitcoin-scripting.cabal

```diff
@@ -12,19 +12,17 @@ category:            Language
 build-type:          Simple
 extra-source-files:  CHANGELOG.md
 
-tested-with: GHC == 8.8.4
+tested-with: GHC == 8.10.4
 
 common base
   default-language: Haskell2010
-  ghc-options:      -Wall
+  ghc-options: -Wall -Wunused-packages -Werror
```

Comment:
There's a bunch of other stuff you could add if you wanted.

```
    - -Wall
    - -Werror
    - -Wmissing-home-modules
    - -Widentities
    - -Wincomplete-uni-patterns
    - -Wincomplete-record-updates
    - -Wpartial-fields
    - -Wmissing-export-lists
    - -fwarn-tabs
    - -fno-warn-unused-do-bind
    - -funbox-strict-fields
    - -Wunused-packages
 ```

that might be a good list

Reply:
Excellent.  I'll add them.

Reply:
Some of these are redundant: https://downloads.haskell.org/ghc/8.10.4/docs/html/users_guide/using-warnings.html#options-sanity

Reply:
(turned out the only one that was redundant was warn tabs!)